router/router.go

```diff
@@ -35,7 +36,10 @@ func Register(
 
 	e.Use(middleware.RequestIDWithConfig(middleware.RequestIDConfig{
 		Generator: func() string {
-			requestId := uuid.New()
+			requestId, err := uuid.NewRandom()
+			if err != nil {
+				time.Now().Format(time.RFC3339Nano)
```

Comment:
haha we need to set the id variable here @guacamole

Reply:
I totally missed to add a return there jeez